### PR TITLE
Add bundled world changes. Clean up out of date info.

### PR DESCRIPTION
**Pull request by dabeshouse**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,34 @@
...
-- Json File - the UI for selecting and reading Json files.
-- Json Zip - the UI for selecting and reading Json zip files.
-- Tweedle Project - the UI for selecting and reading a3w files into a TweedleSystem.
+# Alice Player
+A stand alone program to run worlds exported from the Alice 3 IDE.
+
+Available for desktop on Mac, Windows, and Linux, and for VR on Oculus and Vive.
+
+---
+Build using Unity 2021.3.27f1
```
Comment (jennej) [inline README.md:7]: i'm a unity newb, are there any configuration bits required to get started building, or will this be obvious to people who know what they're about?

File: README.md [hunk 2]
```diff
@@ -1,22 +1,34 @@
...
+## Assets
+
+### _Scenes/Tweedle Project.unity
+The root scene for the project
+
+### _Scripts
+The `Dynamic Import` folder has the code for dynamically importing models both through asset bundles as well as GLTFUtility.
+To create asset bundles from the models the script can be run in Unity (Assets > Build AssetBundles).
+
+The `Tweedle` folder contains code for reading and running Tweedle, the text representation of Alice code. It is in the Java/C# family.
```
Comment (jennej) [inline README.md:18]: maybe add a note about what kinds of changes to other code would require player changes as well?

**Review by jennej — APPROVED** [review]
oooh, formatting

File: README.md (hunk 1 above)
Comment (dabeshouse) [reply README.md:7]: It should be enough to open the project in this version of Unity.
That said, my machine is not a good place to check that, with all the things installed and set up in years gone by. I will be setting up a new machine in the next few weeks and may have cause to update this if I learn of additional steps.
If you try, please let me know of troubles or missed steps.

File: README.md (hunk 2 above)
Comment (dabeshouse) [reply README.md:18]: It is too hard (or I am too lazy) to characterize such changes.
I suppose I could look through the git history to try to pick some examples, but I will instead refer you to the above parenthetical.

**Pull request merged**